pkg/providers/amifamily/resolver.go

```diff
@@ -278,14 +278,7 @@ func (r DefaultResolver) resolveLaunchTemplates(
 	if len(capacityReservationIDs) == 0 {
 		capacityReservationIDs = append(capacityReservationIDs, "")
 	}
-	httpProtocolUnsupportedRegions := sets.New[string](
-		"us-iso-east-1",
-		"us-iso-west-1",
-		"us-isob-east-1",
-		"us-isob-west-1",
-		"us-isof-south-1",
-		"us-isof-east-1",
-	)
+	httpProtocolUnsupportedRegions := awssdk.HTTPProtocolUnsupportedRegions
```

Comment:
nit.

Do we need to declare this variable? Seems like we are only reading this set, so can we just use  `awssdk.HTTPProtocolUnsupportedRegions` for the rest of the function